### DISCUSSION TITLE
Training ASR Fast-Conformer-CTC

### DISCUSSION THREAD

**Discussion by NoorAlaaSaad** [NVIDIA-NeMo/NeMo#8186] [Q&A]
Hello everyone, Currently, I train Fast-Conformer-CTC from scratch for the Arabic Quran Recitation Recognition problem, and I have a problem with long audio samples longer than 12 seconds. Is there any way to make these audios longer than 12 seconds fit into the model without decreasing batch size? I use RTX 3070-Ti, and this is a snap from my config.
![config](https://github.com/NVIDIA/NeMo/assets/125295179/159c2b66-f04c-4732-83da-1a71e1815993)

Any suggestions even if changing model architecture

**Comment by titu1994** [comment]
You can try training with fp16 but apart from that reducing batch size is the main way to avoid oom
